src/sentry/integrations/issues.py

```diff
@@ -31,7 +31,10 @@ def get_group_body(self, group, event, **kwargs):
 
     def get_group_description(self, group, event, **kwargs):
         output = [
-            absolute_uri(group.get_absolute_url()),
+            u'Associated Sentry Issue: [{}]({})'.format(
```

Comment:
is this formatting going to work for jira too?

Reply:
See https://github.com/getsentry/sentry/pull/9295